### ISSUE TITLE
Allow policy statements in Globals 

### ISSUE THREAD

**Issue by brysontyrrell** [awslabs/serverless-application-model#378]
I have been setting up Metrics for a serverless application I'm working on, and it would be very helpful to be able to define some of the policy statements in my Globals and have them **appended** with the policy statements for all of the functions.

Without support for the statements in Globals, I have this block repeated across every Lambda function in my template:

```
          - Effect: Allow
            Action: sqs:SendMessage
            Resource: !GetAtt MetricsQueue.Arn
```

(In my example, I'm aggregating counts using an SQS queue that a separate Lambda will read on a schedule to post data points into CloudWatch Metrics)

**Comment by jfuss** [comment]
@brysontyrrell You got to X-Ray before I could suggest it :)

As far as the rest of the Policies goes, this was an explicit decision by us to not add Policies to the Globals section. I am open to the addition but let me explain why we chose to not add Policies there.

From what I can recall, the biggest reason for this was due to the ease of giving a wider permissions to a function then indented and breaks our least privilege model. I think Policies should be one of the special Properties that should be added to every function, even if it is annoying and makes the template longer.
